content/docs/1.9.0/important-notes/_index.md

```diff
@@ -58,6 +60,28 @@ The `v1beta1` version of the Longhorn API is deprecated in v1.9.0 and will be re
 
 Deprecated APIs are no longer served and may therefore cause unexpected or unwanted behavior. Avoid using longhorn.io/v1beta1 in new code and, if possible, rewrite existing code to exclude this version.
 
+## Breaking Change
+
+### V2 Backing Image
+
+Starting with Longhorn v1.9.0, V2 backing images are incompatible with earlier versions due to naming conflicts in the extended attributes (`xattrs`) used by SPDK backing image logical volumes. As a result, V2 backing images must be deleted and recreated during the upgrade process.
+
+Since backing images cannot be deleted while volumes using them still exist, you must first back up, delete, and later restore those volumes.
+
+#### Upgrade Instructions
+
+- Before upgrading to v1.9.0:
+  - Verify that backup targets are functioning properly.
+  - Create full backups of all volumes that use a V2 backing image.
+  - Detach and delete these volumes after the backups complete.
+  - In the **Backing Image** page, save the specifications of all V2 backing images, including the name and the image source.
+  - Delete all V2 backing images.
+- After upgrading:
+  - Recreate the V2 backing images using the saved name and image source.
+  - Restore the volumes from your backups and allow them to rebuild.
```

Comment:
```suggestion
  - Restore the volumes from your backups and corresponding backing images.
```

Reply:
It does not ask for a backing image while restoring a volume from UI. The backing image is required by the backup rebuilding internally.

Reply:
If it is internal rebuilding, we don't need to mention `allow them to rebuild`.